src/test/java/gov/healthit/chpl/aqa/pageObjects/SearchPage.java

```diff
@@ -22,13 +21,22 @@ public static WebElement acbFiltersLink(final WebDriver driver) {
         return element;
     }
     /**
-     * Returns SLI Compliance ACB filter option.
+     * Returns the certification status filter button.
+     * @param driver webdriver
+     * @return filter status button element
+     */
+    public static WebElement certStatusFiltersButton(final WebDriver driver) {
+        element = driver.findElement(By.id("filter-status-button"));
+        return element;
+    }
+    /**
+     * Returns filter option.
      * @param driver WebDriver
-     * @param acb is acb name
-     * @return SLI Compliance ACB filter option element
+     * @param selectfilter is desired filter name
+     * @return filter option element
      */
-    public static WebElement acbSLIFilter(final WebDriver driver, final String acb) {
-        element = driver.findElement(By.id("filter-list-" + acb));
+    public static WebElement filterOption(final WebDriver driver, final String selectfilter) {
+        element = driver.findElement(By.id("filter-list-" + selectfilter));
```

Comment:
In this POM location, the passed in string should have spaces, and the POM should replace all of the spaces with underscores, something like `"filter-list-" + selectFilter.replaceAll(" ", "_")`